examples/next-starter/src/app/configs/appConfig.ts

```diff
@@ -1,9 +1,21 @@
+import { AppConfig } from '@onr/core';
 import getConfig from 'next/config';
+
 const { publicRuntimeConfig } = getConfig() as {
   publicRuntimeConfig: Record<'processEnv' | string, unknown>;
 };
 
-export const appConfig = {
+export const appConfig: AppConfig = {
```

Comment:
if the `appConfig` is constant and read-only value, I suggest using `as const` make the object cannot be modified.

Reply:
Good point, thanks!